generatebundlefile/ecr_helper.go

```diff
@@ -180,3 +181,15 @@ func getLastestImageSha(details []ImageDetailsBothECR) (*api.SourceVersion, erro
 	}
 	return &api.SourceVersion{Name: latest.ImageTags[0], Digest: *latest.ImageDigest}, nil
 }
+
+// copyImage will copy an OCI artifact from ECR us-west-2 to ECR Public within the same account.
```

Comment:
I wonder about the veracity of this comment, given that "source" and "destination" could be any two (possibly different?) OCI compliant registries, for which "authfile" has access, right?